src/OpenDebugAD7/OpenDebugAD7.csproj

```diff
@@ -17,6 +17,12 @@
     <TargetFramework>net5.0</TargetFramework>
     <Prefer32Bit>false</Prefer32Bit>
 
+    <!--
+      For non-Windows platforms, .NET Core depends on libicu for data about locales and international settings.
+      GDB does not support localization so neither does OpenDebugAD7 or MICore.
+    -->
+    <InvariantGlobalization Condition="!$([MSBuild]::IsOSPlatform('windows'))">true</InvariantGlobalization>
```

Comment:
AFAIK this is just a Linux problem (it would work on macOS). So up to you if you want to make this conditional to `!Windows` or `Linux`